### PR TITLE
Add Helm documentation

### PR DESCRIPTION
**Pull request by lucperkins**
This PR provides documentation for running VK using Helm and also makes some changes to the Helm charts to provide more clear error messages

### CODE REVIEW

**Comment by rbitia** [comment]
Thanks for the updates! LGTM

**Comment by lucperkins** [comment]
@rbitia Don’t merge! This is a WIP

**Comment by lucperkins** [comment]
@rbitia Okay, this is ready for review. It's a bit thin but we can circle back to it later.

File: charts/virtual-kubelet/values.yaml [hunk 1]
```diff
@@ -3,8 +3,6 @@ image:
   tag: latest
   pullPolicy: Always
 
-## `provider` should be one of aws, azure, azurebatch, etc...
```
Comment (rbitia) [inline charts/virtual-kubelet/values.yaml]: Sorry, I might have missed something but why are we taking the provider out of the chart?

File: website/content/docs/usage.md [hunk 2]
```diff
@@ -4,19 +4,19 @@ description: Run a Virtual Kubelet inside or outside of your Kubernetes cluster
 weight: 2
 ---
 
-Virtual Kubelet is run via the `virtual-kubelet` command-line tool (documented [below](#virtual-kubelet-cli)). You can run Virtual Kubelet either [outside](#outside-k8s) or [inside](#inside-k8s) of a Kubernetes cluster.
+You can Virtual Kubelet either [outside](#outside-k8s) or [inside](#inside-k8s) of a Kubernetes cluster using the [`virtual-kubelet`](#virtual-kubelet-cli) command-line tool. If you run Kubernetes inside of a Kubernetes cluster, you can also deploy it using [Helm](#helm).
```
Comment (rbitia) [inline website/content/docs/usage.md]: What does inside vs. outside mean in this context?

Comment (rbitia) [reply website/content/docs/usage.md]: Oh I see, maybe we should say locally instead?

File: charts/virtual-kubelet/values.yaml (hunk 1 above)
Comment (lucperkins) [reply charts/virtual-kubelet/values.yaml]: This doesn't remove the provider. This provides what I personally think is a better error message if you don't specify a provider. The current error message:

```
error calling eq: invalid type for comparison
```

This PR changes the error message to this:

```
You must specify a Virtual Kubelet provider
```

File: website/content/docs/usage.md (hunk 2 above)
Comment (lucperkins) [reply website/content/docs/usage.md]: This language is borrowed from the README so I'm off the hook 😄 I'll take a look and see if I can make that more clear.

Comment (lucperkins) [reply website/content/docs/usage.md]: This language is basically borrowed 1:1 from the README. I definitely agree that this needs clarification. I'll file an issue and address that in another PR.

**Comment by cpuguy83** [comment]
I think this is a good start.
Devil is in the details in setting up providers, unfortunately :(

**Pull request merged**